wordpress-mobile/WordPress-iOS
# What's New - Remove hardcoded values from service calls

Fixes #14584

This removes hardcoded values and passes the actual app version to the endpoint.

To test:

- Go to Me/App Settings/What's New in WordPress
- Make sure that there are three announcement shown, with the same content (title, subtitle and image) as the screenshots below
- The "Find out more" button, at the moment, should open WordPress.com in the default browser (this will change in the future)
- The "Continue" button should dismiss the screen
- Try different devices/iOS versions and orientations and make sure that the views look right

Note: the content of the announcements is purely for testing, and does not reflect actual features of WordPress-iOS

iPhone | iPad
--- | ---
<img height="400" src="https://user-images.githubusercontent.com/34376330/92183713-cb8d1980-ee14-11ea-962a-7c484a3cfcd4.png"> | <img height="400" src="https://user-images.githubusercontent.com/34376330/92183739-d9db3580-ee14-11ea-8224-39fc4e7f7bc4.png">




PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Future PR
- [x] ~~I have considered adding accessibility improvements for my changes.~~ Future PR
